pom.xml

```diff
@@ -178,7 +179,7 @@
         <dependency>
             <groupId>com.residence</groupId>
             <artifactId>residence</artifactId>
-            <version>1.0</version>
```

Comment:
Still the same api ?

Reply:
My bad. Those last two commits were intended for my own development setup. They can be discarded.